src/protocol/resp/src/request/get.rs

```diff
@@ -77,6 +78,19 @@ impl Compose for GetRequest {
     }
 }
 
+impl Klog for GetRequest {
+    type Response = Response;
+
+    fn klog(&self, response: &Self::Response) {
+        let code = match response {
+            Message::BulkString(_) if *response == Response::null() => MISS,
```

Comment:
this is an area where having storage return an enum of possible result conditions would be nice